server/src/main/java/org/eclipse/openvsx/adapter/VSCodeAdapter.java

```diff
@@ -89,7 +91,7 @@
         produces = MediaType.APPLICATION_JSON_VALUE
     )
     @CrossOrigin
-    public ExtensionQueryResult extensionQuery(@RequestBody ExtensionQueryParam param) {
+    public ResponseEntity<ExtensionQueryResult> extensionQuery(@RequestBody ExtensionQueryParam param) {
```

Comment:
This is a POST endpoint. Do the cache control headers have any effect here? The caching proxies would have to consider the whole JSON payload in caching keys.

Reply:
Probably very little. The request body can get quite complex with multiple filters and each filter can have multiple criteria as well. I'll remove the caching for this operation.